docs/download.md

```diff
@@ -12,17 +12,13 @@ bodyclass: download-page
 		<th>Description</th>
 	</tr>
 	<tr>
-		<td><a href="https://leafletjs-cdn.s3.amazonaws.com/content/leaflet/v1.9.4/leaflet.zip">Leaflet 1.9.4</a></td>
+		<td><a href="https://registry.npmjs.org/leaflet/-/leaflet-1.9.4.tgz">Leaflet 1.9.4</a></td>
```

Comment:
Instead of providing the download as tgz via npm, we could use the release zip-asset: 

```suggestion
		<td><a href="https://github.com/Leaflet/Leaflet/releases/download/v1.9.4/leaflet.zip">Leaflet 1.9.4</a></td>
```